pages/api/transactions/[address].ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiResponse, NextApiRequest } from 'next/types'
+import { getAddress } from 'services/bchdService'
+import { RESPONSE_MESSAGES } from 'constants/index'
+
+const { ADDRESS_NOT_PROVIDED_400 } = RESPONSE_MESSAGES
+
+export default async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+  try {
+    const { address } = req.query
+    if (address === '' || address === undefined) {
+      res.status(ADDRESS_NOT_PROVIDED_400.statusCode).send({ message: ADDRESS_NOT_PROVIDED_400 })
+    }
+
+    const response = await getAddress(address)
+    res.status(200).send(response)
+  } catch (e) {
```

Comment:
Here is necessary to type the `e` variable, else typescript complains about it too.